client/lib/plugins/store.js

```diff
@@ -183,22 +183,25 @@ PluginsStore = {
 		if ( sites.length === 0 ) {
 			return [];
 		}
-		sites.forEach( function( site ) {
-			var sitePlugins = PluginsStore.getSitePlugins( site );
+		sites.forEach( ( site ) => {
+			if ( sites.length > 1 && ! site.visible ) {
```

Comment:
We use this method in the site selected case too, in that case we want the list of plugins for that site even when the site is invisible.